### PR TITLE
FIX: Update the "intro" section of the peer review guide

### PR DESCRIPTION
**Pull request by lwasser**
This PR updates and cleans up a lot of content in the About section of the peer review guide. It includes

* A reorganization of content related to peer review overview
* It begins to deprecate the code of conduct in favor of it living in our governance docs pyopensci.org/governance/code-of-conduct [see issue here for more](https://github.com/pyOpenSci/governance/issues/29)
* It adds an explicit benefits of peer review section
* It adds a technical scope criteria to our reviews in the instance that we have a forked repo where upstream isn't independently sourced.

### CODE REVIEW

File: about-peer-review/how-peer-review-works.md
Comment (NickleDave) [inline about-peer-review/how-peer-review-works.md:7]:
```suggestion
contributed by community members with a great diversity of skills and wide range of backgrounds
```
rephrase to avoid repeating "diversity"? I understand the goal here to be to show how we have developers with *both* diversity in skills and backgrounds (which are not the same thing)

File: about-peer-review/pyopensci-related-joss-ropensci.md
Comment (NickleDave) [inline about-peer-review/pyopensci-related-joss-ropensci.md:2]: The structure of this page is a bit confusing to me because it does not start out by defining either JOSS or rOpenSci. After reading further I realize that JOSS is defined below, but I don't think rOpenSci gets mentioned again? If it were me I might structure like so:

```
# How Are JOSS, rOpenSci and pyOpenSci Related?
As mentioned on other pages, pyOpenSci works closely with rOpenSci and JOSS. [rOpenSci](link) is a sister organization that directly inspired pyOpenSci; [The Journal of Open Source Scientific Software(JOSS)](link )is a journal and a publisher that provides a mechanism for developers of research software to get credit for their work within academic systems. The JOSS review process is about publication. A review from JOSS will provide you with a citable DOI that is cross-ref enabled.
pyOpenSci aligns closely with the broad mission of JOSS to provide maintainers with credit for their efforts in developing the open source tools necessary to support reproducible open science workflows.

## How is review at pyOpenSci different from the JOSS review process?
We are not a typical publisher or journal. Rather we are a community that provides support for both a diverse group of software maintainers and long term maintenance of our packages. The pyOpenSci review process is different from that of JOSS in a few ways:

1. Our review is specifically design to enforce modern, community-accepted best practices for Python packaging. 
1. pyOpenSci places heavy emphasis on documentation and usability in our reviews and associated standardization of both.
1. pyOpenSci builds community around and visibility for its tools.
1. We will promote packages and package maintainers once they are accepted into our ecosystem.  
1. We support long term maintenance of packages. If the maintainer needs to step down, we will ensure a new maintainer takes over OR sunset and remove the package from our ecosystem.
1. We provide a welcoming forum to ask questions and get help with maintaining your package as needed.  

JOSS reviews are  also [more limited in scope](https://joss.readthedocs.io/en/latest/review_criteria.html) compared to pyOpenSci and the
[submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html)
are, in places, less stringent than those of pyOpenSci.

## You can improve your package with a review at pyOpenSci and still publish in JOSS
... rest of page here ...
```

File: about-peer-review/pyopensci-related-joss-ropensci.md
Comment (NickleDave) [inline about-peer-review/pyopensci-related-joss-ropensci.md:11]:
```suggestion
## How is pyOpenSci different from JOSS and other review processes?
```
As in that structure I suggested, I'd get rid of two headlines at the top of the page, one right after the other, and instead replace this "Why two review processes" which reads very conversationally with the second headline that says the same thing while better emphasizing the need for different reviews

File: about-peer-review/review-benefits.md
Comment (NickleDave) [inline about-peer-review/review-benefits.md:38]: > [Read more about our partnership with JOSS here](pyopensci-and-joss).
Is this link missing or should it be different text? It doesn't render when I build locally

File: scope.md
Comment (NickleDave) [inline scope.md:31]:
```suggestion
- **Package is an out of sync fork of another package repository.** 
```
Make bold to match other bullet points

File: scope.md
Comment (NickleDave) [inline scope.md:32]:
```suggestion
We understand that sometimes a package maintainer may need to step down from their role. In that case, we strongly suggest that the original package owner transfer the package to a new organization that would allow transfer of ownership of the package rather than 
```
- "Sometimes we understand" sounds like there are times we don't understand
- remove comma splice

Comment (NickleDave) [reply scope.md:32]: I think maybe we don't want to make a blanket statement about "out-of-sync forks" because my guess is it's highly likely people *will* be developing forks precisely *because* the original developers and/or maintainers have stopped updating or even responding to issues, pull requests, etc.

Maybe the criterion should be something like:  
"submissions should not be a fork of a package *that is itself still actively maintained*, that would require maintainers of the fork to have detailed knowledge of both the fork itself and the upstream package. Such a package structure severely limits the number of possible maintainers and contributors. In some cases, this could be a sign that code needs to be vendored, although vendoring code can cause its own complexities (see next bullet)"

File: scope.md
Comment (NickleDave) [inline scope.md:37]:
```suggestion
language or a lower level tool, we prefer that the dependency be added as a 
```

File: scope.md
Comment (NickleDave) [inline scope.md:39]: Maybe this bullet point should be first--maybe it should even be more like an intro paragraph--and all the other ones should follow it? I think 1 and 2 are special cases of this one? What we really want is to avoid complex package structure, and forks of actively maintained packages, as well as packages that vendor a ton of code would both make things complex

**Review by NickleDave — CHANGES_REQUESTED** [review]
The peer review guide is really coming along and I'm excited to see it get to this point. I see where you are going and I hope my review helps you get there a little easier. :rocket: :snake: :heavy_plus_sign: :scientist:

File: about-peer-review/pyopensci-related-joss-ropensci.md
Comment (arianesasso) [inline about-peer-review/pyopensci-related-joss-ropensci.md:40]:
```suggestion
you can also choose to register it with JOSS. JOSS has [more limited scope](https://joss.readthedocs.io/en/latest/review_criteria.html) for packages that it will review. For instance, while pyOpenSci will review and 
```

File: scope.md
Comment (arianesasso) [inline scope.md:32]:
```suggestion
Sometimes we understand that a package maintainer may need to step down. In that case, we strongly suggest that the original package owner transfer the package to a new organization that would allow the transfer of ownership of the package rather than 
```

File: scope.md
Comment (arianesasso) [inline scope.md:29]: I think this paragraph is a little confusing, and there are different mentions of scope here and in "aims and scope". Is it possible to unify those views? I would say that the package is out of scope if it fits one of those criteria + some other technical criteria defined here or elsewhere.

`Your package may not be in scope for us to review at this time if
it fits any of the criteria listed below:

**List of out-of-technical-scope criteria** 

Your python package should also comply with a list of technical criteria that is still being defined.`

File: about-peer-review/aims-and-scope.md
Comment (Batalex) [inline about-peer-review/aims-and-scope.md:32]: The template is well laid out! Would there be a legitimate interest in knowing if the submitting author is committing to the package's maintenance at this point?

File: about-peer-review/how-peer-review-works.md
Comment (Batalex) [inline about-peer-review/how-peer-review-works.md:19]: Not directly related to this file _per se_, is there a plan to maintain a list of potential reviewers based on their specific field of expertise?

File: about-peer-review/intro.md
Comment (Batalex) [reply about-peer-review/intro.md:14]: I guess it refers to the tooling: tests, documentation, linter configuration, CI, etc.

I am +1 on clarifying this bit, though

File: about-peer-review/aims-and-scope.md
Comment (lwasser) [reply about-peer-review/aims-and-scope.md:29]: please look lower down on the page for the technical scope section. it should look like this:

Screen Shot 2022-12-13 at 1 09 28 PM

I could create a link on the page to that section?

Comment (lwasser) [reply about-peer-review/aims-and-scope.md:29]: oh i can see how you wouldn't see that. i have that scope document as an include file to avoid making mistakes / inconsistencies with text around scope given we may want to publish the scope in a few spots. .

File: scope.md
Comment (lwasser) [inline scope.md:25]: @arianesasso this is the scope document with the technical scope listed and it's included in the aims and scope page.

File: about-peer-review/aims-and-scope.md
Comment (lwasser) [reply about-peer-review/aims-and-scope.md:32]: good point. @Batalex do you think we should add that to the presubmission template as a link ? we just want to  avoid people trying to publish a package (a one time activity) and be done vs vet it and maintain over time.

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:8]: got it! i think i follow and updated locally :) your writing is excellent btw!

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:17]: totally confusing. i reworded it as such:

* Editors manage incoming package submissions. they also ensure 
that package reviews move through the review process efficiently; 

is that better?

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:18]: thanks!

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:19]: great question @Batalex we actually have a google sheet already with this type of information. i can share it in our slack with you if you want to have a look. we have people signing up regularly. what i don't love about what we have is that it is a google sheet. i'd rather have a database which is what ropensci has. but that is another discussion.

feel free to ping me with more questions about this because it's a great question and something i've been thinking about in terms of how do we manage this.

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:29]: oh good call. i just find that so many people mix up github and git! so it's habit to write github.com :) but i hear you!

Comment (lwasser) [reply about-peer-review/how-peer-review-works.md:29]: fixed!

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md:13]: sure it's terribly worded. let me rewrite using your suggestion! thank you!

Comment (lwasser) [reply about-peer-review/intro.md:13]: what about something like this?
i broke it into two statements. better? or still confusing?
*******
Test coverage that supports both maintenance of code function. Test coverage also makes it easier for contributors to see how their contributions impacts other parts of the code,

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md:14]: oh totally. yes when i say infrastructure i am talking about CI, make files, linters all that infrastructure that supports the code and evaluates code quality. let me try to clarify here.

Comment (lwasser) [reply about-peer-review/intro.md:14]: what about something like this: (better? or still unclear?)

Evaluation of infrastructure such as continuous integration to run rest suites and check code linters, that supports automated checks on pull requests. This infrastructure supports software quality and reliability.

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md:23]: sure i'll accept your change. one thing i've learned is we don't want to burden screen readers with alt text and captions that say the same thing :) boy is that annoying. but let me accept this and we can see what it looks like in the file!

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md:51]: this is interesting. i'm happy to make this change BUT ... there is a [recent blog ](https://ropensci.org/blog/2022/11/30/first-peer-reviewed-stats-packages/) about how they just started reviewing stats packages. so i get that its thought of that way BUT honestly they started not reviewing those packages and built that capacity out with Mark as the lead in the past 2 years. i actually have met with him to discuss it. so i get it's a commonly used term but not 100% sure they think of themselves that way!

Comment (lwasser) [reply about-peer-review/intro.md:51]: actually let me know what you think about that before i merge this specific change.

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md:100]: yay :) highlighting your amazing work @arianesasso !!

File: about-peer-review/pyopensci-related-joss-ropensci.md
Comment (lwasser) [reply about-peer-review/pyopensci-related-joss-ropensci.md:2]: gosh i've really struggled with this. i decided to make a page about all of the invovled organizations because i thought it was unclear. i think previously i talked about JOSS like 800 times in 800 different places BECAUSE i was worried about people thinking they had to chose between them vs us. Anyway i'll work on this re-org because i think this page is really important and it needs to be clear. i get the joss question ALL OF THE TIME.

Comment (lwasser) [reply about-peer-review/pyopensci-related-joss-ropensci.md:2]: wow it's SO MUCH BETTER. i was struggling because the JOSS content was literally across several pages and i just needed that fresh set of eyes. pushing here in a few but already i think it's majorly improved with this feedback. THANK YOU!

File: about-peer-review/review-benefits.md
Comment (lwasser) [reply about-peer-review/review-benefits.md:38]: good catch! there is a funny sphinx "feature" where if a link that is local is broken it just doesn't render as a link 😆 fixed. the file name was wrong as i had updated it.

File: scope.md
Comment (lwasser) [reply scope.md:29]: i totally can see how this is confusing. but this file is an include file. so it will be rendered on the aims and scope page. i just want to ensure you can see circle ci which does render the file for you. feel free to ask on slack and you can also ask @NickleDave as he is familiar with the ci setup too. i know you are diving in without some of the background context and appreciate the feedback!

File: scope.md
Comment (lwasser) [reply scope.md:39]: ok i've reorganized this and added those bullets as examples. and made the last one a IMPORTANT call out block.

**Comment by lwasser** [comment]
ok @Batalex @arianesasso @NickleDave i've addressed (i think) all of your comments. please feel free to have a look again. i'm going to add a zenodo file to this repo in another PR with your names in it. thank you for all of the great feedback I think it really improved this section of our peer review guidebook

File: about-peer-review/aims-and-scope.md
Comment (Batalex) [reply about-peer-review/aims-and-scope.md:32]: I took the liberty of submitting a PR: https://github.com/pyOpenSci/software-review/pull/66

File: about-peer-review/aims-and-scope.md
Comment (arianesasso) [reply about-peer-review/aims-and-scope.md:29]: Now I get it 😄. Thanks for clarifying. I like the idea of the link too.

File: about-peer-review/intro.md
Comment (arianesasso) [reply about-peer-review/intro.md]: Agree!

File: about-peer-review/intro.md
Comment (arianesasso) [reply about-peer-review/intro.md]: 🤩 I love it! Can I make a little suggestion on top of your suggestion?

`Test coverage that supports the maintenance of code function. Test coverage also makes it easier for contributors to see how their contributions impact other parts of the code,`

Comment (lwasser) [reply about-peer-review/intro.md]: Absolutely. @arianesasso i forget do you know how to suggest the text change directly using the suggest text change button? i forget if you already were doing that so my apologies if you were. Please go ahead and review and then suggest any and all changes in the new files that i pushed last night and I will merge those.  

You can just highlight whatever lines of text you want to change and use that +- button to make changes. i'll accept more changes through next tuesday from everyone! MANY THANKS!! 
Screen Shot 2022-12-14 at 9 21 34 AM

File: scope.md
Comment (NickleDave) [reply scope.md]: @arianesasso I ended up just building locally so I could see the page in context.  

One way to do this is to clone the repo, check out the pull request (e.g. with the GitHub cli you can do `gh pr checkout 148`) and then set up for dev as the README explains so you can run `nox -s docs-live` to get a live build in your local browser. For me this is easier than looking at artifacts on Circle CI.  

Reviewing a PR of a website is not easy compared to code! People must have other tools for this. I also tried the GitHub VS Code extension for reviewing PRs but I found it easier to just use the web interface

File: about-peer-review/aims-and-scope.md
Comment (lwasser) [reply about-peer-review/aims-and-scope.md:29]: ok wonderful!! thanks for all of the great feedback!

File: about-peer-review/intro.md
Comment (lwasser) [reply about-peer-review/intro.md]: excellent! fixed! great feedback resolving now!

File: scope.md
Comment (lwasser) [reply scope.md:4]: no worries! ping me on slack if you don't have access to look at it.

File: scope.md
Comment (lwasser) [reply scope.md]: awesome discussion y'all. thanks David for chiming in! building locally might be the easiest option. i don't use the github CLI. you can also grab instructions for git at the bottom of the pr to grab this branch from my fork!

File: about-peer-review/aims-and-scope.md
Comment (lwasser) [reply about-peer-review/aims-and-scope.md:32]: Thanks! i'll look at that PR today as well. going to focus on this pr first and then your pr in software review! thanks @Batalex i do think it's good to set expectations early. otherwise someone would get to the review and maybe walk away then. it saves us all time to tell them early what we expect :) excellent feedback

**Comment by lwasser** [comment]
oh phew! what a great round of reviews. Thank you everyone! 
I am merging this. YES it has a red X. 

This is there because
3/4 links are in this PR so they don't exist online. There is a way to fix this with html proofer I just haven't had time to implement it yet. The 4th is the code of conduct which is in governance. there is a PR open for it but I need to have the board review and OK it. So that is coming soon. Maybe i'll work on it a bit more today actually to get that moving as it's important.

for now i'm merging this!! yahoo!

**Comment by lwasser** [comment]
> The peer review guide is really coming along and I'm excited to see it get to this point. I see where you are going and I hope my review helps you get there a little easier. 🚀 🐍 ➕ 🧑‍🔬

@NickleDave yay!! i'm glad that you see where this is going! and so appreciative of all your feedback and help !! you're definitely helping me and pyOpenSci get to where we want to be!!

File: about-peer-review/how-peer-review-works.md
Comment (lwasser) [reply about-peer-review/how-peer-review-works.md]: great catch!! your feedback is great and it's really good to have non native speakers reviewing. this document needs to be clear to everyone. THANK YOU!!

**Pull request merged**
